dockerfiles/incr.sh

```diff
@@ -57,6 +57,8 @@ DOCKER_REPO=hpccsystems
 BUILD_TYPE=Debug
 [[ -n ${INPUT_BUILD_TYPE} ]] && BUILD_TYPE=${INPUT_BUILD_TYPE}
 BUILD_THREADS=$INPUT_BUILD_THREADS # If not set, picks up default based on nproc
+BUILD_USER=hpcc-systems
```

Comment:
This change seems unrelated (and I'm not sure it's right - should instances of INPUT_BUILD_USER further down now say BUILD_USER? )

Reply:
yes it should. It is unrelated though.
I do need a way to specify the docker repo, so I can push incremental built images to my docker repo, and use them on Azure, GKE etc. - but will make mods in a separate PR.

Reply:
Correction: the [unrelated] change was about BUILD_USER not REPO. I need that so that when performing a -f (force) build, it picks up from my github user not hpccsystems.
(will deal with in separate PR).